### PR TITLE
update parameters to use coingecko route for ratios

### PR DESCRIPTION
**Pull request by husobee**
### Type of change ( select one )

- [ ] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

File: promotion/claim.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func blindCredsEq(a, b []string) bool {
 		return false
 	}
 	// a and b must have same values in same order
-	for i, _ := range a {
```
Comment (nvonpentz) [inline promotion/claim.go:67]: Is this related?

**Review by nvonpentz — APPROVED** [review]

File: promotion/claim.go (hunk 1 above)
Comment (husobee) [reply promotion/claim.go:67]: Got a linting comment on it when i ran the linter, so fixed.
